app/Code.gs

```diff
@@ -150,38 +150,43 @@ function asciidocHandleFontStyle(text, offset, distinctContent) {
   var isBold = text.isBold(offset);
   var isItalic = text.isItalic(offset);
   var isUnderline = text.isUnderline(offset);
-  var isPlain = !isBold && !isItalic && !isUnderline;
+  var isStrikethrough = text.isStrikethrough(offset);
+  // FIXME: getTextAttributeIndices doesn't split on different fonts,
+  // makeing this almost useless
```

Comment:
I think you can also remove the FIXME comment, I don't remember where we're using `getTextAttributeIndices` ?